src/main/java/frc/robot/commands/GyroTurnCommand.java

```diff
@@ -0,0 +1,86 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import frc.robot.subsystems.drive.Gyroscope;
+import frc.robot.subsystems.drive.Drive;
+
+public class GyroTurnCommand extends CommandBase{
+
+    private Gyroscope gyro;
+    private double turnAngle;
+    private Drive drive;
+    private double newAngle;
+   // private PIDController rotateController;
+
+    public GyroTurnCommand(Gyroscope gyro, double turnAngle, Drive drive, double kP, double kI, double kD){
+
+        this.gyro = gyro;
+        this.turnAngle = turnAngle;
+        this.drive = drive;
+       // rotateController = new PIDController(kP, kI, kD);
+
+        addRequirements(gyro);
+        addRequirements(drive);
+    }
+    @Override
+    public void initialize() {
+        double currentAngle = gyro.getGyroAngle();
+        newAngle = currentAngle + turnAngle;
+        drive.setBrakeMode();
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    @Override
+    public void execute() {
+                double error = newAngle - gyro.getGyroAngle();
+                double speed = -error*(0.0027) *.65;
```

Comment:
All these doubles should be assigned at the top of the class with a comment describing them

Reply:
moved and commented